contributions/contribution.yaml

```diff
@@ -71,7 +71,10 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: '#/components/schemas/Contribution'
+                  anyOf:
+                    - $ref: '#/components/schemas/Contribution'
```

Comment:
Even if the search matches the Talent or Capability, the response of this endpoint is always follow the Contribution schema, right?

Reply:
yup. thats right. the response is a contribution. So yup, no need of the additional schemas.

Reply:
updated

Reply:
Thanks.